packages/vuetify/src/install.ts

```diff
@@ -1,53 +1,44 @@
-import OurVue, { VueConstructor } from 'vue'
-import { VuetifyUseOptions } from 'types'
-import { consoleError } from './util/console'
-
-export function install (Vue: VueConstructor, args: VuetifyUseOptions = {}) {
-  if ((install as any).installed) return
-  (install as any).installed = true
-
-  if (OurVue !== Vue) {
-    consoleError('Multiple instances of Vue detected\nSee https://github.com/vuetifyjs/vuetify/issues/4068\n\nIf you\'re seeing "$attrs is readonly", it\'s caused by this')
+import Vuetify, { VuetifySymbol } from './framework'
+import { App } from 'vue'
+import { GlobalVuetifyPreset } from 'types'
+
+export function install (app: App, args: GlobalVuetifyPreset = {}) {
```

Comment:
Not a fan of `args` here - why not just `preset`?

Reply:
updated naming with d0fb9817a